src/apps/Permissions/EntitySelector.js

```diff
@@ -75,6 +96,14 @@ class EntitySelector extends React.Component {
       items.splice(-1, 0, 'Any account')
     }
 
+    if (includeBurnEntity) {
+      items.splice(-1, 0, 'Burn role')
```

Comment:
We will likely want to come up with something better; e.g. we use "discard" when displaying a role's manager if it's the BURN_ENTITY.

These are concepts we don't really want to expose to users through the entity selector, and we should instead add an action to the management panel, in addition to "remove" action, to "discard" the manager.